src/SFA.DAS.FAT.Application/Shortlist/Services/ShortlistService.cs

```diff
@@ -42,12 +42,12 @@ public async Task<GetShortlistsForUserResponse> GetShortlistsForUser(Guid shortl
         return apiResponse;
     }
 
-    public async Task DeleteShortlistItemForUser(Guid id)
+    public async Task DeleteShortlistItem(Guid id)
```

Comment:
Is there a reason to check for shortlist count after the api call instead of before here? Seems to make sense to save an api call if the shortlist count is zero

Reply:
API call is needed to actually delete the item. Count is managed in the browser.